src/framework/standard/configuration.rs

```diff
@@ -401,7 +402,7 @@ impl Configuration {
     /// client.with_framework(StandardFramework::new().configure(|c| c
     ///     .delimiters(vec![", ", " "])));
     /// ```
-    pub fn delimiters(mut self, delimiters: Vec<&str>) -> Self {
+    pub fn delimiters<T: ::std::string::ToString, It: ::std::iter::IntoIterator<Item=T>>(mut self, delimiters: It) -> Self {
```

Comment:
Particular reason you didn't use just `ToString`/`IntoIterator` instead of their absolute paths here?

Reply:
I forgot to use directly after noticing `ToString` and `IntoIterator` where in the prelude. I'll fix it.